x-pack/plugins/infra/public/components/lens/lens_chart.tsx

```diff
@@ -56,43 +56,38 @@ export const LensChart = ({
 
   const sytle: CSSProperties = useMemo(() => ({ height }), [height]);
 
-  const Lens = (
+  const lens = (
```

Comment:
We can make this a react component and that way we can use it without the `<div>`. wdyt?

Reply:
Ah yeah I see now, It's Ok 👍